tests/libxc/dft-anion/input.dat

```diff
@@ -0,0 +1,121 @@
+memory 1 Gb
+                                           
+molecule h2o {                             
+   -1 2                                    
+   O                                       
+   H 1 1.0                                 
+   H 1 1.0 2 104.5                         
+   units angstrom                          
+   no_reorient                             
```

Comment:
ang is default and `no_reorient` shouldn't actually be needed, right?

Reply:
DFT is not completely invariant to its orientation, wouldn't mind having explicit coordinates in case things shift.